partials/tabs/languages.html

```diff
@@ -2,7 +2,10 @@
 <div class="main-content container">
 
     <div class="col-md-8 col-md-offset-2">
-        <input ng-model="searchText" placeholder="Search for a bear or a language" id="search" type="text" class="validate">
+      <div class="input-field">
```

Comment:
@rahulkde There is no need of adding another `<div>` you could just add the class in the original div ;)